client/src/components/Layout/Layout.tsx

```diff
@@ -24,9 +25,12 @@ export const Layout = ({ children }: Props): JSX.Element => {
         {userId && (
           <>
             <p>
-              <Link className={styles.footerLink} to="/logout">
+              <a
+                href={getPagePath(router, 'logout', {})}
```

Comment:
@ai, are there possibility in the router API which lead us not to use third empty parameter when we do not need parameters for route?

Reply:
Yes. You need to pass types when you create a router

See `interface Routes` in main example https://github.com/nanostores/router